lib/doorkeeper/config.rb

```diff
@@ -198,6 +204,8 @@ def extended(base)
     option :realm,                          default: 'Doorkeeper'
     option :force_ssl_in_redirect_uri,      default: !Rails.env.development?
     option :grant_flows,                    default: %w(authorization_code client_credentials)
+    option :access_token_generator,
+    default: "Doorkeeper::OAuth::Helpers::UniqueToken"
```

Comment:
To keep it consistent with the other lines, even if it's a "long line", I'd align the `default` as the others are.